### ISSUE TITLE
CamelDocumentationProviderTestIT#testJavaClassQuickNavigateInfo is not passing

### ISSUE THREAD

**Issue by rastislavpapp** [camel-tooling/camel-idea-plugin#468]
I've checked out the source code and tried to build the project. All tests pass except the one in the title:

```
junit.framework.ComparisonFailure: 
Expected: ... org.apache.camel:camel-core:2.22.0 ...
Actual: ... org.apache.camel:camel-core:2.23.1 ...
```

As I've yet to explore the project further I'm not quite sure that the fix is as simple as changing the expected camel-core library version in the test html file, or the issue is somewhere else (maybe in my project setup, I'm using IDEA 2018.3.4).

**Comment by fharms** [comment]
I have upgraded the to latest version IDEA and disable that test case for now because it's unstable

After you have updated your project, make sure to install the IntelliJ libraries again

**Comment by fharms** [comment]
fixed in #485
